### PR TITLE
tests: add checks for project_component_updated and

### PR DESCRIPTION
**Pull request by goapunk**
module_component_updated signals

**Describe your changes**
Add checks for the project_component_updated and module_component_updated signals. This should help to discover the usage of them and find bugs

requires https://github.com/liqd/adhocracy4/pull/1533 to work

**Tasks**
- [ ] PR name contains story or task reference
- [ ] Steps to recreate and test the changes
- [ ] Documentation (docs and inline)
- [x] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

File: meinberlin/apps/dev/management/commands/comments.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            "--user",
+            default=0,
+            type=int,
+            help="primary key of user used to create the comments",
+        )
+        parser.add_argument(
+            "--idea",
+            default=-1,
+            type=int,
+            help="primary key of the idea to add the comments to",
```
Comment (m4ra) [inline meinberlin/apps/dev/management/commands/comments.py:30]: this argument is missing from the usage example in line 16. Also it 'd be good to add each argument in the help string.

File: meinberlin/apps/dev/management/commands/comments.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    Usage:
+
+        $ ./manage.py comments --user <pk> --count <int>
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--user",
+            default=0,
```
Comment (m4ra) [inline meinberlin/apps/dev/management/commands/comments.py:22]: are there users with pk 0?

File: meinberlin/apps/dev/management/commands/comments.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        )
+
+    def handle(self, *args, **options):
+        user_pk = options["user"]
+        idea_pk = options["idea"]
+        count = options["count"]
+        user_model = get_user_model()
+        user = user_model.objects.get(pk=user_pk)
+        idea = Idea.objects.get(pk=idea_pk)
+
```
Comment (m4ra) [inline meinberlin/apps/dev/management/commands/comments.py:46]: Need a check if user and idea are > 0.

File: meinberlin/apps/topicprio/views.py [hunk 4]
```diff
@@ -147,6 +147,7 @@ def get_permission_object(self):
 class TopicDeleteView(
     mixins.DashboardBaseMixin,
     mixins.DashboardComponentMixin,
+    mixins.DashboardComponentFormSignalMixin,
     mixins.DashboardComponentDeleteSignalMixin,
```
Comment (m4ra) [inline meinberlin/apps/topicprio/views.py:151]: why keeping both signals?

File: meinberlin/apps/maptopicprio/views.py [hunk 5]
```diff
@@ -143,6 +143,7 @@ def get_permission_object(self):
 class MapTopicDeleteView(
     mixins.DashboardBaseMixin,
     mixins.DashboardComponentMixin,
+    mixins.DashboardComponentFormSignalMixin,
     mixins.DashboardComponentDeleteSignalMixin,
```
Comment (m4ra) [inline meinberlin/apps/maptopicprio/views.py:147]: why keeping both signals?

**Review by m4ra — CHANGES_REQUESTED** [review]

File: meinberlin/apps/dev/management/commands/comments.py (hunk 2 above)
Comment (goapunk) [reply meinberlin/apps/dev/management/commands/comments.py:22]: oh, I added this by mistake to the PR, sorry :/ i used this to test some comment related stuff

Comment (m4ra) [reply meinberlin/apps/dev/management/commands/comments.py]: good to have though. could you make a separate PR and keep it?

File: meinberlin/apps/maptopicprio/views.py (hunk 5 above)
Comment (goapunk) [reply meinberlin/apps/maptopicprio/views.py:147]: my thought was that if we remove it we won't get any warning if there's still js code calling `delete`

File: meinberlin/apps/dev/management/commands/comments.py (hunk 2 above)
Comment (goapunk) [reply meinberlin/apps/dev/management/commands/comments.py]: will do

File: meinberlin/apps/maptopicprio/views.py (hunk 5 above)
Comment (goapunk) [reply meinberlin/apps/maptopicprio/views.py:147]: but I'm also ok with removing it, any preference?

Comment (m4ra) [reply meinberlin/apps/maptopicprio/views.py:147]: OK, but where do we look for the warnings? Only if we keep them in a log file.

Comment (goapunk) [reply meinberlin/apps/maptopicprio/views.py:147]: they should be in the server logs (/var/log/django/django.log)

**Comment by m4ra** [comment]
@goapunk can you do a rebase locally and push again?

**Comment by goapunk** [comment]
> @goapunk can you do a rebase locally and push again?

done

**Review by m4ra — APPROVED** [review]

**Pull request merged**